mephisto/client/review/review_server.py

```diff
@@ -362,6 +365,6 @@ def after_request(response):
     else:
         thread = threading.Thread(target=consume_data)
         thread.start()
-    if sys.stdout.isatty():
```

Comment:
I think we need the above if statement, otherwise when you don't use the `--stdout` flag, e.g. you are piping back out to a file, then it will print out the message into the file too. Were you running with issues with this not showing up in your tests when running them?

Reply:
Ah yes, I must have left it over by accident from trouble shooting. Will add that line back